README.md

```diff
@@ -75,121 +72,131 @@ cd celo-l2-node-docker-compose
+      ```md
+      NODE_TYPE=archive
+      OP_GETH__SYNCMODE=full
+      ```
+
+    This is because nodes will not accept requests for RPC methods that require archive state data unless we set `NODE_TYPE=archive`, and because archive nodes that use snap sync do not store state until the block at which they finish syncing. For these reasons, we recommend using the configurations above. See below for more information.
+
+### Environment Variables
+
+- **NODE_TYPE**
+  - `full` - A full node stores historical state only for recent blocks.
+  - `archive` - An archive node stores historical state for the entire history of the blockchain.
+- **OP_GETH__SYNCMODE** - Sync mode to use for L2 node
+  - `snap` - If left empty, `snap` sync will be used. `snap` sync downloads chaindata from peers until it receives an unbroken chain of headers up through the most recent block, and only begins executing transactions from that point on. Archive nodes that run with `snap` sync will only store state from the point at which the node begins executing transactions.
+  - `full` - `full` sync executes all transactions from genesis (or the last block in the datadir) to verify every header. Archive nodes that run with `full` sync will store state for every block synced, as `full` sync must calculate the state at every block in order to verify every header.
+- **OP_NODE__RPC_ENDPOINT** - Specify the Layer 1 RPC endpoint (e.g., Ethereum mainnet). For instance, you can use the Alchemy free plan for Ethereum mainnet.
+- **OP_NODE__L1_BEACON** - Specify the Layer 1 beacon endpoint. For instance, you can use [QuickNode](https://www.quicknode.com). E.g `https://xxx-xxx-xxx.quiknode.pro/db55a3908ba7e4e5756319ffd71ec270b09a7dce`.
+- **OP_NODE__RPC_TYPE** - Specify the service provider for the RPC endpoint you've chosen in the previous step. The available options are:
+  - `alchemy` - Alchemy
+  - `quicknode` - Quicknode (ETH only)
+  - `erigon` - Erigon
+  - `basic` - Other providers
+- **HEALTHCHECK__REFERENCE_RPC_PROVIDER** - Specify the public healthcheck RPC endpoint for the Layer 2 network.
+- **HISTORICAL_RPC_DATADIR_PATH** - Datadir path to use for legacy archive node to serve pre-L2 historical state.
+- **OP_GETH__HISTORICAL_RPC** - RPC Endpoint for fetching pre-L2 historical state. If set, this overrides the **HISTORICAL_RPC_DATADIR_PATH** setting.
+- **IMAGE_TAG**[...]__ - Use custom docker image for specified components.
+- **PORT**[...]__ - Use custom port for specified components.
+
+## Obtaining a migrated datadir
+
+If you are not using `snap` sync, then a migrated pre-hardfork full node datadir is required. Your options for obtaining one are as follows.
+
+:::warn
+The instructions in this README are are for illustrative purposes only. Please refer to [Migrating a Celo L1 node](https://docs.celo.org/cel2/operators/migrate-node) for the most up to date information.
+:::
```

Comment:
I don't think this is valid gh flavored markdown. Either use plain markdown (e.g. "WARNING: ...") or use [gfm alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts):

 ```
> [!WARNING]
> Urgent info that needs immediate user attention to avoid problems.
```

Using not gh compatible markdown doesn't make sense in this case, as most users will read it on github.